### DISCUSSION TITLE
umi4和umi3如何对src内某一个文件单独打包后放到head里

### DISCUSSION THREAD

**Discussion by wenlei0617** [umijs/umi#11919] [Q&A]
诉求是，期望监控在head里进行加载，尽可能最快初始化。

**Comment by fz6m** [comment]
通过 [father](https://github.com/umijs/father) 等工具将全局前置脚本单独打包为 umd 文件，之后放到 `public` 文件内，这样构建完后就会自动 copy 到产物文件夹了，然后通过配置 [`headScripts`](https://umijs.org/docs/api/config#headscripts) 内联。
